style.css

```diff
@@ -599,10 +636,37 @@ footer ul {
     grid-area: nav;
   }
 
+  .js-nav {
+    background-image: none;
+  }
+
+  nav ul {
+    flex-direction: row;
+    height: 100px;
+    margin: 0 37.22%;
+    padding: 0;
+  }
+
   .burger {
     display: none;
   }
 
+  .cross {
+    display: none !important;
+  }
+
+  .deco-a {
+    display: none;
+  }
+
+  .deco-b {
+    display: none;
+  }
+
+  .mos-0 {
+    display: none;
+  }
```

Comment:
- [ ] I would recommend to follow the DRY (Do not repeat yourself) principle to follow the CSS best practices. Here you have gave the same property to three different classes so you can choose something like this.

```
.class-a,
.class-b,
.class-c {
   ...... // any property
 }
```

Reply:
Request satisfied in last commit.

Reply:
This issue was well resolved! ✔️